### DISCUSSION TITLE
Your RBAC solutions

### DISCUSSION THREAD

**Discussion by francoisauclair911** [open-policy-agent/community#48] [OPA and Rego]
Hi everyone!

I'm starting out with OPA / Rego and I'm having some issue finding examples of RBAC solution implemented with Rego (apart from the one in the official playground).
I thought I'd open this discussion to encourage people to post their Rego playground URL in here.

Maybe the following template could be used

**Title: 
Description: 
Playground URL:** 

Cheers

**Comment by francoisauclair911** [comment]
**Title:**  RBAC using grants
**Description:** Example found in the rego playground list of examples
**Playground URL:**  [https://play.openpolicyagent.org/p/3xSPetsM0s](https://play.openpolicyagent.org/p/3xSPetsM0sI)

**Reply by peteroneilljr** [reply]
ohh awesome, thanks for starting this thread @francoisauclair911 !!

**Comment by francoisauclair911** [comment]
**Title:**  RBAC with relationship
**Description:** RBAC for users with multiple roles in multiple organizations (tenant)
**Playground URL:**  [https://play.openpolicyagent.org/p/wL8BgtVXAQ](https://play.openpolicyagent.org/p/wL8BgtVXAQ)
